neuromation/api/parser.py

```diff
@@ -64,3 +108,61 @@ def _remote_to_local_image(self, image: RemoteImage) -> LocalImage:
             self._config.username, self._config.cluster_name, self._config.registry_url
         )
         return parser.convert_to_local_image(image)
+
+    def env(
+        self, env: Sequence[str], env_file: Sequence[str] = ()
+    ) -> Tuple[Dict[str, str], Mapping[str, URL]]:
```

Comment:
Why `Tuple[Dict[str, str], Mapping[str, URL]]`, but not `Tuple[Dict[str, str], Dict[str, URL]]` or `Tuple[Mapping[str, str], Mapping[str, URL]]`?